Sources/TuistGenerator/Models/DeploymentTarget.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+// MARK: - DeploymentTarget
+
+public enum DeploymentTarget {
+    case iOS(String, [DeploymentDevice])
+    case macOS(String)
+    // TODO: 🙈 Add `watchOS` and `tvOS` support
```

Comment:
What platforms should we support in this PR? 🤔

Reply:
We don't have proper support for watchOS and tvOS yet so I'd leave those commented out.